src/main/java/hudson/plugins/build_timeout/operations/AbortOperation.java

```diff
@@ -65,6 +65,8 @@ public boolean perform(AbstractBuild<?, ?> build, BuildListener listener, long e
         if (e != null) {
             e.interrupt(Result.ABORTED);
         }
+        build.setResult(Result.ABORTED);

```

Comment:
The result set with `e.inturrupt` isn't used as you suppressed `InterruptedException`.
I don't think it's good idea to suppress `InterruptException` as it may cause Jenkins work differently when you use `BuildTimeOutWrapper` and `BuildStepWithTimeout`.

As far as I tested the behavior, stacktraces aren't  output into build logs (console logs), but are output only into system logs.
Can I know more details about why you want to suppress `InterruptedException`?